shell/imports/db-deprecated.js

```diff
@@ -54,62 +54,3 @@ matchWildcardHost = globalDb.matchWildcardHost.bind(globalDb);
 makeWildcardHost = globalDb.makeWildcardHost.bind(globalDb);
 allowDevAccounts = globalDb.allowDevAccounts.bind(globalDb);
 roleAssignmentPattern = globalDb.roleAssignmentPattern;
-
-if (Meteor.isServer) {
-  getWildcardOrigin = globalDb.getWildcardOrigin.bind(globalDb);
-
-  Meteor.onConnection((connection) => {
-    // TODO(cleanup): This is the best way I've thought of so far to allow methods declared in
-    //   packages to actually use the DB, but it's pretty sad.
-    connection.sandstormDb = globalDb;
-  });
-  SandstormDb.periodicCleanup(5 * 60 * 1000, SandstormPermissions.cleanupSelfDestructing(globalDb));
-  SandstormDb.periodicCleanup(10 * 60 * 1000,
-                              SandstormPermissions.cleanupClientPowerboxRequests(globalDb));
-  SandstormDb.periodicCleanup(24 * 60 * 60 * 1000, () => {
-    SandstormAutoupdateApps.updateAppIndex(globalDb);
-  });
-
-  Meteor.startup(() => { globalDb.migrateToLatest(); });
-  LDAP_DEFAULTS.url = globalDb.getLdapUrl();
-  LDAP_DEFAULTS.base = globalDb.getLdapBase();
-}
-
-if (Meteor.isClient) {
-  Session.setDefault("shrink-navbar", false);
-  globalGrains = new GrainViewList(globalDb);
-
-  // If Meteor._localStorage disappears, we'll have to write our own localStorage wrapper, I guess.
-  // Using window.localStorage is dangerous because it throws an exception if cookies are disabled.
-  Session.set("shrink-navbar", Meteor._localStorage.getItem("shrink-navbar") === "true");
-  globalTopbar = new SandstormTopbar(globalDb,
-    {
-      get() {
-        return Session.get("topbar-expanded");
-      },
-
-      set(value) {
-        Session.set("topbar-expanded", value);
-      },
-    },
-    globalGrains,
-    {
-      get() {
-        return Session.get("shrink-navbar");
-      },
-
-      set(value) {
-        Meteor._localStorage.setItem("shrink-navbar", value);
-        Session.set("shrink-navbar", value);
-      },
-    });
-
-  globalAccountsUi = new AccountsUi(globalDb);
-
-  Template.registerHelper("globalTopbar", () => { return globalTopbar; });
-  Template.registerHelper("globalAccountsUi", () => { return globalAccountsUi; });
-} else {
-  // TODO(cleanup): Refactor accounts registration stuff so that this doesn't need to be defined
```

Comment:
As far as I can tell, this has already happened.  I grepped the codebase for `globalAccountsUi` and found no references in code that wasn't explicitly under `/client`.